### DISCUSSION TITLE
Need help in Bandwidth used in a particular time

### DISCUSSION THREAD

**Discussion by hcodes07** [grafana/grafana#66779] [Data Sources]
Hi I am using influxdb as source and getting the data from librenms, I am able to create graph for bandwidth utilization for network devices. I really need help in getting the total bandwidth data used a particular time with Each for download and upload.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
